codecov.yml

```diff
@@ -1,13 +1,29 @@
-comment: false
 coverage:
   status:
     project:
       default:
         target: auto  # auto compares coverage to the previous base commit
         threshold: 10%  # this allows a 10% drop from the previous base commit coverage
         informational: true
+
 ignore:
   - "tests"
   - "sentry_sdk/_types.py"
+
+# Read more here: https://docs.codecov.com/docs/pull-request-comments
+comment:
+  # This is the addition of carry forward builds and fresh builds, thus, it's the addition
+  # of the FE and BE builds
```

Comment:
Not sure what this comment means

Reply:
oh, copy pasta from `sentry`. will remove